content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -157,9 +157,8 @@ volumeMounts:
   - mountPath: /etc/kubernetes/audit-policy.yaml
     name: audit
     readOnly: true
-  - mountPath: /var/log/audit.log
+  - mountPath: /var/log
```

Comment:
If only mounting a single file will cause log rotation errors, should we add precautions to let users mount directories instead of files as much as possible?

Reply:
So many users will use the examples on github directly. We should ensure our examples can be used robustly.
If only add notes, user maybe ignore these notes.

Reply:
Do you means we should not only change our mount path, but also should add some notes?  Nice suggestion.

Reply:
> If only mounting a single file will cause log rotation errors, should we add notes to let users mount directories instead of files as much as possible?

I added some notes on line 172. please review it